org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/namedobjects/NamedObjectInheritanceTest.xtend

```diff
@@ -19,7 +19,7 @@ class NamedObjectInheritanceTest extends AbstractWollokInterpreterTestCase {
 			
 			program p {
 				assert.equals("abc", myObject.something())
-				assert.equals(myObject, (myObject->"42").getX())
+				assert.equals(true, (myObject == myObject))
```

Comment:
I added it on order to avoid ambiguities with  `->` in closures. 

Again, I nearly found no uses of `->` in the code base. Why is having literal pairs - and not is general concept of tuples - so important?

Anyway, another option is to use `=>` for creating tuples, or `=>` for closures. I prefer the first option, since it is far more common to use closures than tuples. What do you think?